public/images/sidebar-icon.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="248" height="136" viewBox="0 0 248 136" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I'd move this right next to the sidebar component. The complication figures out static assets
The path can be imported like [this](https://github.com/SeedCompany/cord-field/blob/0d1b88608ba583fad4e9d7dcccec545c1664ebf1/src/scenes/Authentication/Authentication.tsx#L7)

----

I'm also seeing this rendered now too. It has the CORD Icon and some text as well as the background. I'm wondering if it would just be better to layer the cord icon and text on top to reduce this svg size and keep the text styling consistent with the theme (font, etc.).

Reply:
I split this up so I can reuse the cord icon. Text and svg is layered on top of that background svg